spec/jobs/socure_shadow_mode_proofing_job_spec.rb

```diff
@@ -0,0 +1,384 @@
+                  attributes_requiring_additional_verification: [],
+                  can_pass_with_additional_verification: false,
+                  errors: {},
+                  exception: nil,
+                  reference: '',
+                  success: true,
+                  timed_out: false,
+                  transaction_id: '',
+                  vendor_name: 'ResidentialAddressNotRequired',
+                  vendor_workflow: nil,
+                  verified_attributes: nil,
+                },
+                resolution: {
+                  attributes_requiring_additional_verification: [],
+                  can_pass_with_additional_verification: false,
+                  errors: {},
+                  exception: nil,
+                  reference: 'aaa-bbb-ccc',
+                  success: true,
+                  timed_out: false,
+                  transaction_id: 'resolution-mock-transaction-id-123',
+                  vendor_name: 'ResolutionMock',
+                  vendor_workflow: nil,
+                  verified_attributes: nil,
+                },
+                state_id: {
+                  errors: {},
+                  exception: nil,
+                  mva_exception: nil,
+                  requested_attributes: {},
+                  success: true,
+                  timed_out: false,
+                  transaction_id: 'state-id-mock-transaction-id-456',
+                  vendor_name: 'StateIdMock',
+                  verified_attributes: [],
+                },
+                threatmetrix: {
+                  client: nil,
+                  errors: {},
+                  exception: nil,
+                  review_status: 'pass',
+                  success: true,
+                  timed_out: false,
+                  transaction_id: 'ddp-mock-transaction-id-123',
+                },
+              },
+            },
+            exception: nil,
+            ssn_is_unique: true,
+            threatmetrix_review_status: 'pass',
+            timed_out: false,
+          },
+          socure_result: {
+            attributes_requiring_additional_verification: [],
+            can_pass_with_additional_verification: false,
+            errors: { reason_codes: ['I905', 'I914', 'I919'] },
+            exception: nil,
+            reference: '',
+            success: true,
+            timed_out: false,
+            transaction_id: 'a1234b56-e789-0123-4fga-56b7c890d123',
+            vendor_name: 'socure_kyc',
+            vendor_workflow: nil,
+            verified_attributes: %i[address first_name last_name phone ssn dob].to_set,
+          },
+        )
+      end
+
+      context 'when socure proofer raises an error' do
+        before do
+          allow(job.proofer).to receive(:proof).and_raise
+        end
+
+        it 'does not squash the error' do
+          # The Proofer converts errors raised during requests into results
+          # with the `exception` property set. Any other error
```

Comment:
Nit: I'm confused by this comment and specifically the trailing "Any other error" bit.